### PR TITLE
Add google analytic display on dashboard

### PR DESCRIPTION
**Pull request by glmanhtu**
- Add some module display data from google analytic to dashboard
- Using google batch to reduce time load for google analytic

### CODE REVIEW

File: core/common/factory/PageViews.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+/**
+ * This class declare TopDashboard Factory
+ */
+class PageViews extends TopDashboard
+{
+    /**
+     * Override function create
+     * Change special detail for each dimension
+     * @param  string $dimension google analytic metric
+     * @return
```
Comment (ghost) [inline core/common/factory/PageViews.php:24]: @return mixed

File: core/common/factory/TopDashboard.php [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+ *
+ * @link    http://phanbook.com Phanbook Project
+ * @since   1.0.0
+ * @license http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt
+ */
+namespace Phanbook\Factory;
+
+class TopDashboard implements TopDashboardInterface
+{
+    public $analytic;
```
Comment (duythien) [inline core/common/factory/TopDashboard.php:25]: Dockblock comment

File: core/common/factory/TopDashboardFactory.php [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+        return $obj;
+    }
+    public function executeBatch()
+    {
+        $result = $this->analyticClient->batchExecute();
+        $this->analyticClient->setUseBatch(false);
+        return $result;
+    }
+}
+
```
Comment (duythien) [inline core/common/factory/TopDashboardFactory.php]: Remove space

File: core/config/routes.php [hunk 4]
```diff
@@ -139,9 +139,6 @@
     if ($uri == '/') {
         return false;
     }
-    if ($uri == '/posts') {
```
Comment (duythien) [inline core/config/routes.php:142]: Why remove it?

File: .travis.yml [hunk 5]
```diff
@@ -20,8 +20,8 @@ env:
   global:
     - APPLICATION_ENV="travis"
   matrix:
-    - PHALCON_VERSION="2.0.8"
-    - PHALCON_VERSION="2.0.7"
+    - PHALCON_VERSION="2.0.6"
+    - PHALCON_VERSION="2.0.5"
```
Comment (sergeyklay) [inline .travis.yml:25]: Why

Comment (duythien) [inline .travis.yml:25]: I don't know. Now commit again :)

**Pull request merged**